api/scenario_api_utils/package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
```

Comment:
Can you add `<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>` to match the ROS 2 format? Thanks.

Reply:
done.